slices/libstd-rust-1.85.yaml

```
@@ -0,0 +1,20 @@
+package: libstd-rust-1.85
+
+essential:
+  - libstd-rust-1.85_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+      - libllvm19_libs
+      - libstdc++6_libs
+    contents:
+      # symlink to ../rust-1.85/lib/librustc_driver-*.so
+      /usr/lib/*-linux-*/librustc_driver-*.so:
```

I've seen these kind of symlink comments in other slices and PRs. As an idea, we could try to put some convention in place for better readability, just something simple:


```suggestion
      /usr/lib/*-linux-*/librustc_driver-*.so: # -> ../rust-1.88/lib/librustc_driver-*.so
```